modules/components/pages/redpanda/about.adoc

```diff
@@ -53,58 +92,85 @@ seed_brokers:
 
 === `logs_topic`
 
-The topic to which logs will be sent.
+The topic logs are sent to.
 
 *Type*: `string`
 
-*Default*: `__redpanda.connect.logs`
+*Default*: `""`
+
+```yml
+# Examples
+
+logs_topic: __redpanda.connect.logs
+```
 
 === `logs_level`
 
-The log level for the logs sent to Redpanda. Possible values are `debug`, `info`, `warn`, `error`. Default is `info`.
+The logging level of logs sent to Redpanda.
 
 *Type*: `string`
 
 *Default*: `info`
 
+*Options*: `debug`, `info`, `warn`, `error`
+
+=== `status_topic`
+
+The topic status updates are sent to.
+
+*Type*: `string`
+
+*Default*: `""`
+
+```yml
+# Examples
+
+status_topic: __redpanda.connect.status
+```
+
 === `client_id`
 
 An identifier for the client connection.
 
-
 *Type*: `string`
 
-*Default*: `"benthos"`
+*Default*: `benthos`
```

Comment:
Here and in a few other cases, values are given in quotes but are they necessary? I have removed as not shown in YAML specification.

Reply:
That's fine